Hide refreshing spinners on select views

drewvolz opened the pull request:
Closes #1710 
Closes #1711
Closes #1712

- Hide refresh spinner for Important Contacts list
- Hide refresh spinner for Dictionary list
- Hide refresh spinner for Bus schedules
- Hide Loading view on pull-to-refresh for Student Orgs list

drewvolz committed:
Remove dictionary refresh spinner

drewvolz committed:
Remove contacts refresh spinner

drewvolz committed:
Remove import from dictionary

drewvolz committed:
Remove bus refresh spinner

drewvolz committed:
Remove orgs loading view while refreshing

drewvolz committed:
Prettier change to contact-list

hawkrives commented on source/views/student-orgs/list.js:
```diff
@@ -192,7 +192,7 @@ export class StudentOrgsView extends React.Component {
 
   render() {
-    if (this.state.loading) {
+    if (this.state.loading && !this.state.refreshing) {
```
Instead of this, you'll want to remove these lines rom fetchData:

https://github.com/StoDevX/AAO-React-Native/blob/b8531c8c710e9ed5d44e159fbcec1ce747a3af87/source/views/student-orgs/list.js#L98

I can do this in a bit, since I think you're indisposed tonight.

drewvolz replied:
That would be appreciated, thanks!

hawkrives committed:
mimic calendar's refreshing behavior in student-orgs

That is, the view is loading by default. Every time getData runs, it sets "loading" to false,
which is a noop after the first render.
